storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -7169,26 +7167,27 @@ bool xb_init() {
 
   /* sanity checks */
   if (opt_lock_ddl && opt_lock_ddl_per_table) {
-    msg("Error: --lock-ddl and --lock-ddl-per-table are mutually exclusive. "
-        "Please specify --lock-ddl=false to use --lock-ddl-per-table.\n");
+    xb::error()
+        << "--lock-ddl and --lock-ddl-per-table are mutually exclusive. "
+           "Please specify --lock-ddl=false to use --lock-ddl-per-table.";
     return (false);
   }
 
   if (opt_slave_info && opt_no_lock && !opt_safe_slave_backup) {
-    msg("Error: --slave-info is used with --no-lock but "
-        "without --safe-slave-backup. The binlog position "
-        "cannot be consistent with the backup data.\n");
+    xb::info() << "--slave-info is used with --no-lock but "
+                  "without --safe-slave-backup. The binlog position "
+                  "cannot be consistent with the backup data.";
     return (false);
   }
 
   if (opt_rsync && xtrabackup_stream_fmt) {
-    msg("Error: --rsync doesn't work with --stream\n");
+    xb::info() << "--rsync doesn't work with --stream";
```

Comment:
Again, I'm unaware of the context here. What were errors before, are info now. Is it intentional?

Reply:
mistake, will fix it